src/Notifaika/Discourse.hs

```diff
@@ -66,11 +66,12 @@ resultToEither :: Result a -> Either String a
 resultToEither (Success s)  = Right s
 resultToEither (Error e)    = Left e
 
-getDiscourseEvents :: (MonadIO m, MonadThrow m) => Url -> m [Event]
+getDiscourseEvents :: (SetMember Lift (Lift IO) r) => Url -> Eff r [Event]
 getDiscourseEvents baseUrl = do
     let url = baseUrl <> "/latest.json"
-    response <- liftIO $ Wreq.get url
-    jsonResponse <- asJSON response
+    jsonResponse <- lift $ do
+      response <- Wreq.get url
+      asJSON response -- XXX: because 'asJSON' require MonadThrow
```

Comment:
Ops because of that we lift both `Wreq.get` and `asJSON` together, previously those methods were separated.